Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.h

```diff
@@ -422,6 +423,9 @@ namespace AzToolsFramework
             // Handler of the public Prefab requests.
             PrefabPublicRequestHandler m_prefabPublicRequestHandler;
 
+            // Handles the Prefab Focus API that determines what prefab is being edited.
+            PrefabFocusHandler m_prefabFocusHandler;
```

Comment:
Prefab system component should be agnostic to editor features like focus mode. It is used by spawnables and proc prefabs and other systems too. So, adding the focus handler here isn't a good idea imo. I think this should either be moved to PrefabPublicHandler or PrefabEditorEntityOwnerhipService